### DISCUSSION TITLE
scripting [Jul 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2615] [Discord Threads]
> 🙋‍♂️ **Kosta the Russian**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### scripting is not yet available.   
####    
#### We've webhooks which can trigger on an update  
#  
> 🙋‍♂️ **Kosta the Russian**  
#### yep, saw that (just installed on Heroku or whatever it's called).. my whole AirTable app is built on scripting.. so will be giving this a pass until scripting is implemented in some way or there is another way around it - a lot of potential though and will keep an eye out chrs  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/991971705985712198)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
